api/src/Microsoft.Azure.IIoT.Api/src/Publisher/Models/EngineConfigurationApiModel.cs

```diff
@@ -48,5 +48,12 @@ public class EngineConfigurationApiModel {
         [DataMember(Name = "maxOutgressMessages", Order = 4,
             EmitDefaultValue = false)]
         public int? MaxOutgressMessages { get; set; }
+
+        /// <summary>
+        /// Flag to use reversible encoding for messages
+        /// </summary>
+        [DataMember(Name = "reversibleEncoding", Order = 5,
```

Comment:
Should we use `useReversibleEncoding` instead of `reversibleEncoding` to be consistent and to easier find places where it is used ?

Reply:
Renamed in #1694.